### ISSUE TITLE
ProtoTypeMap - Outer class incorrect if protobuf name is not lower underscore

### ISSUE THREAD

**Issue by pelidan** [salesforce/grpc-java-contrib#163]
Hi,

the ProtoTypeMap mapping is incorrect when the protobuf file is named in a different convention.

how to reporoduce:

OuterClass.proto
```
syntax = "proto3";

package pkg;

message InnerClass {
    optional int32 value = 1;
}
```

ProtoTypeMapTest:
```
@Test
public void nestedTypeCamelCase() {
    assertProtoTypeMapping(".pkg.InnerClass", pkg.OuterClass.InnerClass.class);
}
```

[ERROR]   ProtoTypeMapTest.nestedTypeCamelCase:68->assertProtoTypeMapping:112 expected:<"pkg.Outer[C]lass.InnerClass"> but was:<"pkg.Outer[c]lass.InnerClass">

cause:
ProtoTypeMap:124 - this produces a different outer class name than the code that generates the Java classes

**Comment by pelidan** [comment]
The only solution that comes to my mind right now is checking whether it is a lower underscore format and skipping the conversion if not.
A better one can exist though.

**Comment by pelidan** [comment]
created PR: https://github.com/salesforce/grpc-java-contrib/pull/164

**Comment by rmichela** [comment]
Good catch. 👀

**Comment by pelidan** [comment]
thanks, it's the root cause why we can't upgrade to Vert.x 4 (vertx-grpc), it took a while to find 😄

**Comment by rmichela** [comment]
Fixed in 1.2.0
